packages/form-js-editor/package.json

```diff
@@ -44,6 +44,7 @@
   "dependencies": {
     "@bpmn-io/form-js-viewer": "^0.9.9",
     "array-move": "^3.0.1",
+    "didi": "^9.0.0",
```

Comment:
Perhaps it should go to devDependencies as we use it for types only: https://github.com/bpmn-io/form-js/blob/56f33dba5a6826aaef6b59d81f65c3b68bd78f77/packages/form-js-viewer/src/util/injector.js#L1